BenchBnb/BenchBnb/Views/Review/Index.cshtml

```diff
@@ -0,0 +1,7 @@
+﻿
+@{
+    ViewBag.Title = "Index";
+}
+
+<h2>Index</h2>
```

Comment:
I'd remove this view and the associated action method in the `ReviewController` class.

Reply:
Fixed